[v5] `createEmptyActor()` (#3968)
statelyai/xstate: .changeset/odd-pigs-brush.md

---
'xstate': major
---

> The `createNullActor()` function has been added to make it easier to create actors that do nothing ("null" actors). This is useful for testing, or for some integrations such as `useActor(actor)` in `@xstate/react` that require an actor:

Reviewer: ```suggestion
The `createEmptyActor()` function has been added to make it easier to create actors that do nothing ("empty" actors). This is useful for testing, or for some integrations such as `useActor(actor)` in `@xstate/react` that require an actor:
```

```jsx

> import { createNullActor } from 'xstate';

Reviewer: ```suggestion
import { createEmptyActor } from 'xstate';
```

const SomeComponent = (props) => {
  // props.actor may be undefined

>   const [state, send] = useActor(props.actor ?? createNullActor());

Reviewer: ```suggestion
  const [state, send] = useActor(props.actor ?? createEmptyActor());
```
